### DISCUSSION TITLE
Getting socket error when using web3 provide packed with wallet connect.

### DISCUSSION THREAD

**Discussion by SolutionPr** [WalletConnect/walletconnect-monorepo#3181] [Developer Support]
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/93113701/37fbbe11-d789-4986-99c7-ab75c87a5e7a)
Please have a look into this I'm using bsc network to get connect. 
mode of connection via cdns 

https://unpkg.com/web3@latest/dist/web3.min.js
https://unpkg.com/web3modal
https://unpkg.com/@walletconnect/web3-provider
https://unpkg.com/fortmatic@2.0.6/dist/fortmatic.js

**Comment by glitch-txs** [comment]
v1 has been depricated, I'd suggest upgrading to [Web3Modal v2](https://docs.walletconnect.com/2.0/advanced/migration-from-v1.x/overview)
